api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -519,12 +519,25 @@ function networkAccountReducer(account) {
     name:
       account && account.identity && account.identity.display
         ? account.identity.display
-        : account.accountId,
+        : account.accountId || '',
     address: account && account.accountId ? account.accountId : '',
     picture: account ? account.twitter : '' // TODO: get the twitter picture using scriptRunner
   }
 }
 
+function getProposalSummary(type) {
```

Comment:
why are those duplicated?

Reply:
which ones?

Reply:
ah, you meant the `getProposalSummary` function. Yeah, better like you did, definitely